### DISCUSSION TITLE
[CSV Import] How to get Localized Fields?

### DISCUSSION THREAD

**Discussion by pento-thal** [pimcore/pimcore#9064] [Q&A / Support] [answered]
I'm trying to upload a bunch of object belonging to a class composed by Localized Fields:
![image](https://user-images.githubusercontent.com/83424908/117484120-f19c2800-af66-11eb-92be-2ce656f2b7c7.png)

I have this type of .csv:
![image](https://user-images.githubusercontent.com/83424908/117484358-45a70c80-af67-11eb-9277-8f770e00f11f.png)

How should I proceed with column configuration to get each translated value?
![image](https://user-images.githubusercontent.com/83424908/117484615-9d457800-af67-11eb-99bb-459b899605cb.png)

**Comment by ghost** [comment]
Hi @pento-thal,
Check this maybe it will help you --> https://talk.pimcore.org/t/export-csv-with-localized-fields/772

**Reply by pento-thal** [reply]
Hi @rushit-credencys, thanks for the reply.
Unfortunately, the page you posted is a discussion regarding a csv **export**.
I need, on the contrary, to understand how to **import** different languages' values for the same attribute.

**Comment by dvesh3** [comment, marked as answer]
@pento-thal just use locale switcher https://pimcore.com/docs/pimcore/6.9/User_Documentation/DataObjects/CSV_Import/Operators/LocaleSwitcher.html
![image](https://user-images.githubusercontent.com/5137917/117843466-1f42e300-b27f-11eb-9b44-71dbd3f76c06.png)

**Reply by pento-thal** [reply]
@dvesh3 thanks, that's exactly what I was looking for!
